src/main.js

```diff
@@ -66,11 +66,11 @@ function onXhrOpenCalled(xhr, method, url) {
         // this is only required if the "gcr=[countrycode]" flag is set in the googlevideo-url...
 
         // solve CORS errors by preventing YouTube from enabling the "withCredentials" option (not required for the proxy)
-        Object.defineProperty(xhr, "withCredentials", {
-            set: () => { },
-            get: () => false,
-        });
+        // Object.defineProperty(xhr, "withCredentials", {
+        //     set: () => { },
+        //     get: () => false,
+        // });
```

Comment:
This doesn't seem like it has effect. Can you confirm?

Reply:
this is only required if the account proxy was used. test it here: https://www.youtube.com/watch?v=gKVjWNv9V8Q
for me it is necessary.
![image](https://user-images.githubusercontent.com/59258980/132967373-53628278-2d49-4277-b328-c300fbfe027a.png)

Reply:
The comment above it had me confused because it says "not required for proxy"

Reply:
yes, a bit complicated xd
This means that the withCredentials option is not needed for the proxy. Therefore setting the withCredentials option will be prevented by setting a descriptor that always returns false. This will also solve the CORS problems :)

Reply:
Okay just remove the "not" from the comment which makes more sense ;)